### PR TITLE
listen for last waypoint reached

### LINKED ISSUE #204: Add support for MISSION_ITEM_REACHED message
See https://groups.google.com/forum/#!topic/drones-discuss/JSstez2glaI

### PR DESCRIPTION
**Pull request by squilter**
Fixes #204 .  Tested and works.

### CODE REVIEW

File: ClientLib/src/main/java/com/o3dr/services/android/lib/drone/attribute/AttributeEvent.java [hunk 1]
```diff
@@ -69,6 +69,7 @@
     public static final String MISSION_SENT = PACKAGE_NAME + ".MISSION_SENT";
     public static final String MISSION_RECEIVED = PACKAGE_NAME + ".MISSION_RECEIVED";
     public static final String MISSION_ITEM_UPDATED = PACKAGE_NAME + ".MISSION_ITEM_UPDATED";
+    public static final String MISSION_ITEM_REACHED = PACKAGE_NAME + ".MISSION_ITEM_UPDATED";
```
Comment (ne0fhyk) [inline ClientLib/src/main/java/com/o3dr/services/android/lib/drone/attribute/AttributeEvent.java]: @squilter the value for the `MISSION_ITEM_REACHED` constant is the same as the one for the `MISSION_ITEM_UPDATED` constant.

**Comment by ne0fhyk** [comment]
@squilter i just merged some code that seems to break your pr. could you update with the latest from `develop`.

**Comment by squilter** [comment]
It was mavlink.jar again.  I will submit a pr to gitignore that after this is merged.

**Comment by squilter** [comment]
@ne0fhyk mission_item_reached is only supported in AC master right now

**Comment by ne0fhyk** [comment]
@squilter this is totally fine, as it's not a critical feature for now.

**Pull request merged**